NoMansLand/Assets/Scripts/Player/CharacterControllerMovement.cs

```diff
@@ -13,14 +13,14 @@ public class CharacterControllerMovement : MonoBehaviour
 
     // Move
     private Vector3 move;
-    private float speed = 50f;
+    private float speed = 85f;
     private float turnSmoothTime = 0.1f;
     private float turnSmoothVelocity;
     
     // Jump
     private Vector3 playerVelocity;
     private float jumpHeight = 5f;
-    private float gravity = -9.81F;
+    private float gravity = -39.81F;
```

Comment:
Not sure why these constants are changed. This is related to player jumping and movement which are fine on controller.

Reply:
For our current terrain which is pretty open, our current player speed and gravity are too low, which makes the game feel more sluggish and tedious when it takes forever to move from one area to another. With the debug script I was enjoying the game more when it had a faster pace and less floaty gravity, so I decided to apply those changes here to see what yall think. So how did you find the updated speed and gravity?